arch/platform/cooja/Makefile.cooja

```diff
@@ -72,5 +72,7 @@ CLEAN += COOJA.log
 CFLAGSNO = $(EXTRA_CC_ARGS) -Wall -g -I/usr/local/include -DCLASSNAME=$(CLASSNAME)
 ifeq ($(WERROR),1)
 CFLAGSNO += -Werror
+CFLAGSNO += -Wno-error=unused-function -Wno-error=format-security
```

Comment:
Why the change? Especially `-Wno-format-security` and `-Wno-error=format-security`. I think it should be left as is.
Which code caused you problems to build without these flags?

Reply:
as i remember printf with only one fmt, whitout arguments, causes this warning